model/lifecycle.go

```diff
@@ -1364,13 +1364,19 @@ func getContainerOptions(project *Project, pRef *ProjectRef, container string) (
 			return &opts, nil
 		}
 
-		size, ok := pRef.ContainerSizes[c.Size]
-		if !ok {
+		var containerSize *ContainerResources
+		for _, size := range pRef.ContainerSizeDefinitions {
+			if size.Name == c.Size {
+				containerSize = &size
+				break
+			}
+		}
+		if containerSize == nil {
 			return nil, errors.Errorf("container size '%s' not found", c.Size)
 		}
 
-		opts.CPU = size.CPU
-		opts.MemoryMB = size.MemoryMB
+		opts.CPU = containerSize.CPU
```

Comment:
should these be set in the if statement on like 1371?

Reply:
These two lines need to be performed after we ensure the container size that we retrieve is not nil, so that's why I chose to put these at the end of the function.